### PR TITLE
Run tests faster! (#507)

### PR DESCRIPTION
**Pull request by michaelhagedon**
Or rather, run multiple tests at once.

This uses [Fastest](https://github.com/liuggio/fastest) to parallelize PHPUnit test classes.

## Description
Modify the probo build to use Fastest to run more jobs at once. Also provide a new lando command:

```
lando phpunit-fastest
```

to run all the tests locally.

## Related Issue
az-digital/az_quickstart/issues/507

## How Has This Been Tested?
Details including terminal output are on the issue. In summary, I've run this many times locally and multiple times in Probo (much faster in Probo!)

Additionally, I verified that these parallel test runs won't conflict because the Drupal test runner automatically prefixes database tables for each test.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by michaelhagedon** [comment]
This is still a draft and is now purposefully breaking until az-digital/az-quickstart-dev/issues/11 is complete.

File: .lando.yml [hunk 1]
```diff
@@ -57,6 +57,9 @@ tooling:
   phpunit:
     service: appserver
     cmd: /app/vendor/bin/phpunit --configuration web/profiles/custom/az_quickstart/phpunit.xml.dist --bootstrap web/core/tests/bootstrap.php web/profiles/custom/az_quickstart
+  phpunit-fastest:
```
Comment (joeparsons) [inline .lando.yml:60]: Would be good to add a ddev command for this as well.  Our ddev config needs some overall improvement but we've been trying to remain consistent about adding any custom tooling commands to both (see https://github.com/az-digital/az_quickstart/blob/main/.ddev/commands/web/phpunit for an example)

Comment (michaelhagedon) [reply .lando.yml:60]: Ah, thanks, I've added that, though I haven't been able to test it yet. Getting ddev going has been a bit challenging, mostly for reasons not related to ddev.

**Comment by joeparsons** [comment]
Triggered ProboCI rebuild...

**Review by joeparsons — APPROVED** [review]
Thanks for working on this Mike.  Tests running in < 3 minutes on ProboCI is amazing!

**Review by trackleft — APPROVED** [review]

**Review by mmunro-ltrr — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Pull request merged**